backend/app/readers/create_readers.py

```diff
@@ -0,0 +1,30 @@
+from app.models.settings import Settings
+from app.readers.reader.reader import Reader
+from app.readers.reader.wikipedia_reader import WikipediaReader
+
+
+def create_readers(settings: Settings) -> tuple[Reader | None, ...]:
+    """
+    Return a tuple of Readers depending on the record type in settings.
+    Readers are instantiated with Paths that contain the corresponding record types in the Path names.
+    """
+
+    readers = []
+    if settings.record_types:
```

Comment:
This is going to be an empty sequence for the null, so you can just iterate over it without checking for `None`.

Reply:
I have removed the null check from `create_readers`.